flight_feed_operations/views.py

```diff
@@ -295,16 +294,17 @@ def start_opensky_feed(request):
 def set_signed_telemetry(request):
     # This endpoint sets signed telemetry details into Flight Blender, use this endpoint to securly send signed telemetry information into Blender, since the messages are signed, we turn off any auth requirements for tokens and validate against allowed public keys in Blender.
 
-    my_message_verifier = MessageVerifier()
-    my_response_signer = ResponseSigningOperations()
-    verified = my_message_verifier.verify_message(request)
+    message_verifier = MessageVerifier()
+    response_signer = ResponseSigner()
+    verified = message_verifier.verify_message(request)
 
     if not verified:
-        message_verification_failed_response = MessageVerificationFailedResponse(
-            message="Could not verify against public keys setup in Flight Blender"
-        )
         return JsonResponse(
-            asdict(message_verification_failed_response),
+            json.dumps(
+                {
+                    "message": "Could not verify against public keys setup in Flight Blender"

```

Comment:
may be "message": "Could not verify against public keys of USSP client(GCS) setup in Flight Blender"